src/compiler/diagnosticMessages.json

```diff
@@ -3519,6 +3519,14 @@
         "category": "Error",
         "code": 7036
     },
+    "Create namespace objects for ECMAScript imports which are neither callable nor constructable, implies `allowSyntheticDefaultImports`.": {
+        "category": "Message",
+        "code": 7037
+    },
+    "Import is called or constructed, which is not valid ES2015 module usage, and will fail at runtime.": {

```

Comment:
`A namespace import can not be called or constructed. Did you mean 'import {0} from "{1}"'?`

Reply:
@DanielRosenwasser thoughts about this one too.

Reply:
The quick fix provides the `Did you mean 'import {0} from "{1}"'` in the quick fix suggestion text. Should that be duplicated? That would effectively mean replicating the entire contents of the quickfix inside the checker.

Reply:
I think that is fine. the message will have the `Did you mean ...`. the quickfix will have a text of `Add ..`. we do that for things like uncalled decorator or the spelling susggession